### ISSUE TITLE
Level ups do not work quite right on latest version of pathfinder. 

### ISSUE THREAD

**Issue by Ozaii** [Ironxxx/pf1ultimatepsionics#7]
I am getting an error of "Could not find class association: Undefined" on the Ultimate Psionics classes from the Module. This is on level up. It did work right before I did the update. This is on the pathfinder version 9.0 and foundry version 11.0. It simply isn't adding the feats on level up like it used to. 

![error on pf1 foundry vtt ultimate psionics](https://github.com/Ironxxx/pf1ultimatepsionics/assets/69611486/0c669cda-d0b0-470d-8228-af398401c928)

**Comment by Ozaii** [comment]
just saw this as well.

The module "pf1ultimatepsionics" is using "name" which is deprecated in favor of "id"
The module "pf1ultimatepsionics" is using the old flat core compatibility fields which are deprecated in favor of the new "compatibility" object

**Comment by Burg973** [comment]
The current module is broken for v11 due to how FoundryVTT updated things in the backend. I've created a fork that updates the module to work in v11, hopefully it is merged and fixes this for everyone.

**Comment by Ozaii** [comment]
> The current module is broken for v11 due to how FoundryVTT updated things in the backend. I've created a fork that updates the module to work in v11, hopefully it is merged and fixes this for everyone.

Thanks bro. I have just one question. How do I get the new json from the fork to work. I replaced the old json with yours but that didn't work at all. Would simple text replace be the way on notepad++?

**Comment by Burg973** [comment]
> > The current module is broken for v11 due to how FoundryVTT updated things in the backend. I've created a fork that updates the module to work in v11, hopefully it is merged and fixes this for everyone.
> 
> Thanks bro. I have just one question. How do I get the new json from the fork to work. I replaced the old json with yours but that didn't work at all. Would simple text replace be the way on notepad++?

yes, I used Notepad++ to edit the original file to get it to work. You should be able to copy and paste it over the original file and save it. I would also try deleting the old module.json and put the new copy in its place. Either method, I recommend then reboot foundry afterwards.

**Comment by Burg973** [comment]
> > The current module is broken for v11 due to how FoundryVTT updated things in the backend. I've created a fork that updates the module to work in v11, hopefully it is merged and fixes this for everyone.
> 
> Thanks bro. I have just one question. How do I get the new json from the fork to work. I replaced the old json with yours but that didn't work at all. Would simple text replace be the way on notepad++?

I believe they should be merged now, you should be able to update the module and it will work now.
